mlflow/ml-package-versions.yml

```diff
@@ -257,7 +257,7 @@ spacy:
   package_info:
     pip_release: "spacy"
     install_dev: |
-      pip install spacy-nightly
+      pip install git+https://github.com/explosion/spaCy
```

Comment:
Switched to installing from GitHub since it looks like `spacy-nightly` hasn't been updated for a while:
https://pypi.org/project/spacy-nightly/#history

Reply:
With this change, the version of spacy will change to `3.0.0rc5` to `3.0.6` ([see this](https://github.com/explosion/spaCy/blob/master/spacy/about.py#L3)), which should fix the issue.

Reply:
It worked: https://github.com/mlflow/mlflow/pull/4389/checks?check_run_id=2662886297

Reply:
Wonderful! Thanks @harupy !